xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -159,10 +202,20 @@
         <Parameter Name="assemblyName" Type="System.Reflection.AssemblyName" />
       </Parameters>
       <Docs>
-        <param name="assemblyName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="assemblyName">The object that describes the assembly to be loaded.</param>
```

Comment:
I would say something like: "An <ref>AssemblyName</ref> instance describing the assembly to be loaded."